src/nrc/tests/commands/files/setup_config_full.yaml

```diff
@@ -0,0 +1,37 @@
+zgw_consumers_config_enable: True
+zgw_consumers:
+  services:
+  - identifier: autorisaties-api
+    label: Objecttypen API test
+    api_root: http://localhost:8001/autorisaties/api/v1/
+    api_type: ac
+    auth_type: zgw
+    client_id: open-notificaties
+    secret: oPMsHCEuoP9Qh8vP06D7
+    user_id: open-notificaties
+    user_representation: Open Notificaties Demodam
+
+autorisaties_api_config_enable: True
+autorisaties_api:
+  # Configure Open Notificaties to make use of Open Zaak's Autorisaties API
+  authorizations_api_service_identifier: autorisaties-api
+
+vng_api_common_credentials_config_enable: True
+vng_api_common_credentials:
+  items:
+  # Credentials for Open Zaak to be able to make requests to Open Notificaties
+  - identifier: open-zaak
+    secret: G2LIVfXal1J93puQkV3O
+
+# notifications_config_enable: True
+# notifications_config:
+#   # No notifications_api_service necessary, because Open Notificaties doesn't send
+#   # notifications to itself
+#   notification_delivery_max_retries: 5
+#   notification_delivery_retry_backoff: 3
+#   notification_delivery_retry_backoff_max: 30
```

Comment:
```suggestion
# notifications_config_enable: True
# notifications_config:
#   # No notifications_api_service necessary, because Open Notificaties doesn't send
#   # notifications to itself
#   notification_delivery_max_retries: 5
#   notification_delivery_retry_backoff: 3
#   notification_delivery_retry_backoff_max: 30
```
This doesn't serve any purpose I guess if it is not possible to configure it

Reply:
I had these in because I initially wanted to refactor everything at once, but I decided to split it up because this relies on the PR in notifications-api-common.

I'll remove it for now